.github/workflows/nodejs.yml

```diff
@@ -94,6 +94,12 @@ jobs:
         if: matrix.webpack-version == '4'
         run: npm i webpack@${{ matrix.webpack-version }} --save-dev --ignore-scripts
 
+      - name: Setup firefox
+        if: matrix.os != 'windows-latest'
+        uses: browser-actions/setup-firefox@latest
```

Comment:
This action doesn't work with windows (https://github.com/browser-actions/setup-firefox/issues/252). But fortunately, our tests are not failing on windows.